apps/README.md

```diff
@@ -189,6 +189,52 @@ fetching files served by the test runner, prefix the file path with
 document.write('<audio src="/base/test/audio/assets/win.mp3"/>');
 ```
 
+#### UI Component Style Guide ####
+
+We use `react-storybook` to generate a ui component style guide that you can use
+to discover what components are available to reuse as you build new
+features. You can also use the style guide to more easily develop new components
+without having to run all of code.org.
```

Comment:
I think deeply nested components can appear here. If you think your component is relevant outside of your app, then I suppose it would be good to move it to a more common place, but that wouldn't be related to your usage of storybook.